pkg/grpc/scheduler/client/client.go

```diff
@@ -45,8 +45,28 @@ func (c *Client) EligibleNodes(man manifest.Manifest, sel klabels.Selector) ([]t
 	return ret, nil
 }
 
-func (c *Client) AllocateNodes(manifest manifest.Manifest, nodeSelector klabels.Selector, allocationCount int) ([]types.NodeName, error) {
-	return nil, util.Errorf("AllocateNodes() not yet implemented")
+func (c *Client) AllocateNodes(man manifest.Manifest, nodeSelector klabels.Selector, allocationCount int) ([]types.NodeName, error) {
+	manifestStr, err := man.Marshal()
+	if err != nil {
+		return nil, util.Errorf("could not marshal manifest for AllocateNodes gRPC request: %s", err)
+	}
+	req := &scheduler_protos.AllocateNodesRequest{
+		Manifest:       string(manifestStr),
+		NodeSelector:   nodeSelector.String(),
+		NodesRequested: int64(allocationCount),
```

Comment:
nit: thoughts on changing the `allocationCount` param name to `nodesRequested`? It'll be consistent with the proto message and `allocationCount` did confuse me in my last PR. I thought it might refer to the total allocations desired. You're choice

Reply:
yeah i think `nodesRequested` will be clearer